Sources/Fuzzilli/Lifting/ScriptWriter.swift

```diff
@@ -14,18 +14,50 @@
 
 struct ScriptWriter {
     /// How many spaces to use per indention level.
-    public static let indent = 4
+    public let indent: Int
     
     /// The current script code.
     var code = ""
     
     /// The current number of spaces to use for indention.
-    private var indention: Int = 0
+    private var currentIndention: Int = 0
+
+    private let minifyOutput: Bool
+
+    public init (minifyOutput: Bool, indent: Int = 4) {
+        self.minifyOutput = minifyOutput
+        self.indent = minifyOutput ? 0 : indent
+    }
+
+    /// If minify mode is turned on, remove whitespaces around characters
+    /// which cannot occur in JS variable names
+    mutating func emitFormattedLine<S: StringProtocol>(_ line: S) {
+        guard self.minifyOutput else {
```

Comment:
Nit: can you make this an `if` instead of a `guard`, since this doesn't just return now but actually does "useful" things

Reply:
When I browsed swift basics, they used some simple logic in guard before returning 🤷‍♂️ , but sure we can follow this convention.

Reply:
Thanks! Yeah I prefer using guards only for cases where the code directly returns or throws an exception and maybe logs an error message before (i.e. for failure cases or exceptional cases)